paddle/fluid/operators/conv_op.cc

```diff
@@ -55,7 +55,7 @@ void ConvOp::InferShape(framework::InferShapeContext* ctx) const {
   std::vector<int64_t> output_shape({in_dims[0], filter_dims[0]});
   for (size_t i = 0; i < strides.size(); ++i) {
     PADDLE_ENFORCE(in_dims[i + 2] + 2 * paddings[i] -
-                           (dilations[i] * (filter_dims[i + 2] - 1) + 1) >
+                           (dilations[i] * (filter_dims[i + 2] - 1) + 1) >=
```

Comment:
It seems there is no need to check here, the check should be in `ConvOutputSize` function to make the returned value valid.

Reply:
Thanks, I refined this code.